### PR TITLE
Newsletter 77:Translate into Japanese

### PR DESCRIPTION
**Pull request by Ichiro0219**

### CODE REVIEW

**Comment by bitschmidty** [comment]
@Ichiro0219 Can you rebase this on master? Seems we have some extraneous commits in this branch.

**Comment by Ichiro0219** [comment]
> @Ichiro0219 Can you rebase this on master? Seems we have some extraneous commits in this branch.
Rebase done.

File: _posts/ja/newsletters/2019-12-18-newsletter-ja.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange] [bitcoin.se]はOptech Contributor達が疑問に対して答えを探しに（もしくは他のユーザーの質問に答える時間がある場合に）アクセスする、数少ない情報ソースです。この月刊セクションでは、前回アップデート以降にされた、最も票を集めた質問・答えについて共有しています。*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Lightningネットワークのパス長制限（20ホップ）の理論的根拠は？]({{bse}} 92073)
```
Comment (bitschmidty) [inline _posts/ja/newsletters/2019-12-18-newsletter-ja.md]: space after {{bse}} should not be there

File: _posts/ja/newsletters/2019-12-18-newsletter-ja.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  Sergei TikhomirovがBOLT4の20ホップ制限とTorのオニオンルーティングとLNのスフィンクスの違いについて質問しています。 Rene Pickhardtは、プロトコルの違いと現在の20ホップの根拠を説明しています。20ホップの根拠については、TCP/ IPパッケージを小さく保つ必要があり、またLN自体が小規模なネットワークである（20ホップもあれば相手に到達できる規模である）という説明しています。
+
+- [トランザクションの構築で未コンファーム状態のRBF(Rplace-by-Fee)アウトプットを使用できるようにする方法はありますか？]({{bse}} 92164)
```
Comment (bitschmidty) [inline _posts/ja/newsletters/2019-12-18-newsletter-ja.md]: extra space

File: _posts/ja/newsletters/2019-12-18-newsletter-ja.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  G. Maxwellは、Bitcoin Coreは、RBFサポートを通知しないトランザクションのアウトプットを処理するのと同じ方法で、RBFを選択したことを通知するトランザクションのアウトプットを処理することを説明しています。 Bitcoin Coreによるアウトプットの処理方法で違いが発生するのは、出力を含むトランザクションがコンファームされたかどうか、およびトランザクションがユーザーのBitcoin Coreウォレットによって作成されたかどうかによるものです。
+
+- [BIP32派生パスに許可される最大の深さは？]({{bse}} 92056)
```
Comment (bitschmidty) [inline _posts/ja/newsletters/2019-12-18-newsletter-ja.md]: extra space

File: _posts/ja/newsletters/2019-12-18-newsletter-ja.md (hunk 1 above)
Comment (Ichiro0219) [reply _posts/ja/newsletters/2019-12-18-newsletter-ja.md]: Fix done.

File: _posts/ja/newsletters/2019-12-18-newsletter-ja.md (hunk 2 above)
Comment (Ichiro0219) [reply _posts/ja/newsletters/2019-12-18-newsletter-ja.md]: Fix done.

File: _posts/ja/newsletters/2019-12-18-newsletter-ja.md (hunk 3 above)
Comment (Ichiro0219) [reply _posts/ja/newsletters/2019-12-18-newsletter-ja.md]: Fix done.

**Comment by Ichiro0219** [comment]
@bitschmidty I fixed all of them and rebased.

**Pull request merged**